harness/tests/storage/test_storage_base.py

```diff
@@ -1,10 +1,13 @@
 import os
+from typing import Optional, Tuple
+from unittest.mock import patch
```

Comment:
please follow the [google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports)

Reply:
What's the problem here? Both lines seem to follow the guide (they mention `typing` as an exception).

Reply:
typing is an exception, I'm talking about `from unittest.mock import patch`.  `typing` aside, the symbol you import should always be a module.  But `patch` is a function, not a module:

```
>>> import unittest.mock.patch
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'unittest.mock.patch'; 'unittest.mock' is not a package
```

So the correct way to import and reference `patch` would be:
```python
from unittest import mock
...
with mock.patch(...):
    ...
```